stac_fastapi/core/stac_fastapi/core/core.py

```diff
@@ -372,22 +372,18 @@ def _return_date(interval_str):
         Returns:
             dict: A dictionary representing the date interval for use in filtering search results.
         """
-        intervals = interval_str.split("/")
-        if len(intervals) == 1:
-            datetime = f"{intervals[0][0:19]}Z"
-            return {"eq": datetime}
+        intervals = interval_str
+        if type(intervals) != tuple:
+            return {"eq": intervals}
         else:
             start_date = intervals[0]
             end_date = intervals[1]
-            if ".." not in intervals:
-                start_date = f"{start_date[0:19]}Z"
-                end_date = f"{end_date[0:19]}Z"
-            elif start_date != "..":
-                start_date = f"{start_date[0:19]}Z"
+            if None not in intervals:
+                ...
```

Comment:
`pass` is preferred to `...`. People sometimes use `...` for empty functions, but i prefer pass in all instances as its clearer.

Reply:
Updated as suggested